go/test/endtoend/tabletgateway/vtgate_test.go

```diff
@@ -66,7 +66,15 @@ func TestVtgateReplicationStatusCheck(t *testing.T) {
 	qr := utils.Exec(t, conn, "show vitess_replication_status like '%'")
 	expectNumRows := 2
 	numRows := len(qr.Rows)
-	assert.Equal(t, expectNumRows, numRows, fmt.Sprintf("wrong number of results from show vitess_replication_status. Expected %d, got %d", expectNumRows, numRows))
+	assert.Equal(t, expectNumRows, numRows, "wrong number of results from show vitess_replication_status")
+
+	qr = utils.Exec(t, conn, "select max_repl_lag()")
+	expectNumRows = 1
+	numRows = len(qr.Rows)
+	assert.Equal(t, expectNumRows, numRows, "expected single row from `select max_repl_lag()`")
+
+	_ = utils.Exec(t, conn, "use @spare")
+	utils.AssertMatches(t, conn, `select max_repl_lag()`, `[[NULL]]`)
```

Comment:
I would make the function name `max_replication_lag` (repl fully spelled out instead of contracted)

Reply:
I think we can name it `vt_max_replication_lag` so it does not interfere with any MySQL functions.